### ISSUE TITLE
[BUG] Attempted to add foreign key to table without id field

### ISSUE THREAD

**Issue by deanlj** [TiagoSilvaPereira/vemto-issues#120]
```
TypeError: Cannot read property 'id' of undefined
    at a.setSelectedEntity (app://./js/project_schema.1eb9a046.js:1:28285)
    at a.refresh (app://./js/project_schema.1eb9a046.js:1:27597)
    at a.save (app://./js/project_schema.1eb9a046.js:1:28938)
    at a.<anonymous> (app://./js/project_schema.1eb9a046.js:1:28730)
    at m (app://./js/chunk-vendors.a7383248.js:74:49773)
    at b (app://./js/chunk-vendors.a7383248.js:74:50037)
    at v (app://./js/chunk-vendors.a7383248.js:74:49989)
```

---
 
vemto 0.8.2
Electron 8.5.5
darwin 18.7.0
Locale: en-US

**Comment by TiagoSilvaPereira** [comment]
@deanlj thank you for reporting, I'll fix it ASAP

**Comment by TiagoSilvaPereira** [comment]
Fixed on the version 0.9.2
